.github/workflows/find-unused.yml

```
@@ -0,0 +1,42 @@
+name: CI - Find Unused Exports, Types and Files
+env:
+  NODE_OPTIONS: '--max_old_space_size=6144'
+on:
+  push:
+    branches-ignore:
+      - 'renovate/**'
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  lint:
+    name: Find Unused
+    runs-on: ubuntu-20.04
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [16.x, 18.x]
```

```suggestion
        node-version: [16.x]
```

We are not there yet, it will be updated later as a part of #48117